### PR TITLE
Use figure assets from published book

### PR DESCRIPTION
**Pull request by chrishtr**
This PR does the following:

* Adds the assets from the [online version](https://academic.oup.com/book/59224) to our website, replacing hand-drawn versions.
* Makes some figures as visible on the web that were not previously.
* Fixes a bug in the JS for resizing iframes.
* Fixes a broken link for the zoom CSS property.
* Fixes line wrapping for one example of the source code.

### CODE REVIEW

File: www/book.css [hunk 1]
```diff
@@ -212,7 +214,7 @@ img { max-width: 100%; }
 .installation:before { content: "Installation"; color: blue; }
 .demo { outline: 2px solid gray; }
 .demo:before { content: "Demonstration"; color: gray; }
-.further { outline: 2px solid green; padding-top: 0; }
```
Comment (chrishtr) [inline www/book.css:215]: I'm not sure why this rule was there before - do you remember?

File: book/text.md [hunk 2]
```diff
@@ -262,7 +262,8 @@ of the text by setting the `anchor` argument to `"nw"`, meaning the
 x, y = 200, 225
 canvas.create_text(x, y, text="Hello, ", font=font1, anchor='nw')
 x += font1.measure("Hello, ")
-canvas.create_text(x, y, text="overlapping!", font=font2, anchor='nw')
+canvas.create_text(
+    x, y, text="overlapping!", font=font2, anchor='nw')
```
Comment (pavpanchekha) [inline book/text.md:266]: A little ugly but I guess  it's not that important.

**Review by pavpanchekha — APPROVED** [review]

**Comment by pavpanchekha** [comment]
Do we 100% have rights to these images? Like, I guess so, OUP said we were supposed to get them.

**Comment by chrishtr** [comment]
> Do we 100% have rights to these images? Like, I guess so, OUP said we were supposed to get them.

Yes, definitely. They said yes to giving us the assets.

**Pull request merged**